### PR TITLE
jit egm-lecture (new PR)

### PR DESCRIPTION
**Pull request by natashawatkins**
copy of #182

### CODE REVIEW

**Comment by jstac** [comment]
Thanks @natashawatkins 

Is this ready for review?

File: source/rst/egm_policy_iter.rst [hunk 1]
```diff
@@ -130,161 +140,211 @@ Finally, for each :math:`c_i` we set :math:`y_i = c_i + k_i`
...
 
+Here's an implementation of :math:`K` using EGM as described above
 
+Unlike the :doc:`previous lecture <coleman_policy_iter>`, we do not just-in-time
+compile the operator because we want to return the policy function
 
-Testing on the Log / Cobb--Douglas case
-------------------------------------------
+Despite this, the EGM method is still faster than the standard Coleman-Reffett operator,
+as we will see later on
```
Comment (natashawatkins) [inline source/rst/egm_policy_iter.rst]: @jstac It seems we decided to go with returning the function, and therefore not jitting, and it was just as fast. I added this discussion here

Comment (jstac) [reply source/rst/egm_policy_iter.rst]: Roger that, sounds good.

**Comment by natashawatkins** [comment]
@jstac yep. All three optimal growth lectures were updated because we use the same files across the lectures, and also for consistency.

**Comment by jstac** [comment]
Thanks @natashawatkins , nice work.  Merging.

**Pull request merged**